model/registry/deprecated/http.yaml

```diff
@@ -31,11 +31,11 @@ groups:
         examples: ['/search?q=OpenTelemetry#SemConv']
       - id: request_content_length
         type: int
-        brief: 'Deprecated, use `http.request.body.size` instead.'
+        brief: 'Deprecated, use the opt-in `http.request.header.content-length` instead.'
```

Comment:
Here, we are in the registry where the requirement level is sort of meaningless. Why do we need that `... the opt-in ...` addition here?